test/e2e/lib/pages/gutenboarding/designs-page.js

```diff
@@ -11,11 +10,15 @@ export default class DesignLocatorPage extends AsyncBaseContainer {
 
 	async selectFreeDesign() {
 		const freeOptions = await this.driver.findElements( this.freeOptionLocator );
-		await freeOptions[ dataHelper.getRandomInt( 0, freeOptions.length - 1 ) ].click();
-	}
+		if ( freeOptions.length === 0 ) {
+			throw new Error( 'No free designs were found on design selection page' );
+		}
+
+		if ( freeOptions.length === 1 ) {
+			throw new Error( 'There should be more than one design found on the design selection page' );
+		}
 
-	async selectPaidDesign() {
-		const paidOptions = await this.driver.findElements( this.paidOptionLocator );
-		await paidOptions[ dataHelper.getRandomInt( 0, paidOptions.length - 1 ) ].click();
```

Comment:
The paid option isn't used anywhere currently, and we are migrating to playwright, so I removed it so that it doesn't incidentally get used in the meantime and cause similar behavior with the randomness.